dockerfiles/theia-dev/build-scripts/default/01-install-dependencies.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+#
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+apk add --update --no-cache \
+  curl `# Download some files` \
+  jq `# To play with json in shell` \
+  make gcc g++ python `# compile some javascript native stuff (node-gyp)` \
+  git openssh openssh-keygen `# clone repositories (and also using ssh repositories)` \
+  less `# Handle git diff properly` \
+  bash `# bash shell` \
+  libx11-dev libxkbfile-dev `# some lib to compile 'native-keymap' npm mpdule`
+
```

Comment:
do we need to install yarn here, like in https://github.com/eclipse/che-theia/pull/467/files#diff-b601fdde7ca9d45a0d2f10471ca7bfdf ?

Reply:
no we don't, yarn is already installed on community images this is why I haven't installed it

Reply:
https://github.com/nodejs/docker-node/blob/a9c583095d4cf08bbd68f570a1f9a99780820351/10/alpine/Dockerfile#L61